cst438_register/src/test/java/com/cst438/EndToEndAddStudentTest.java

```diff
@@ -0,0 +1,155 @@
+package com.cst438;
```

Comment:
Really great testing.  Very straightforward and well commented.  Fantastic PR.

Reply:
Thanks.